libraries/AP_Proximity/AP_Proximity.h

```diff
@@ -61,6 +61,7 @@ class AP_Proximity
         AirSimSITL = 12,
 #endif
         CYGBOT_D1 = 13,
+        LEDDAR_VU8 = 14,
```

Comment:
This needs to be added to the PRX_TYPE metadata

Reply:
fixed in latest push